oshmem/proc/proc.h

```diff
@@ -71,7 +58,7 @@ struct oshmem_group_t {
     int                         my_pe;
     int                         proc_count;     /**< number of processes in group */
     int                         is_member;   /* true if my_pe is part of the group, participate in collectives */
-    struct oshmem_proc_t      **proc_array;     /**< list of pointers to ompi_proc_t structures
+    struct ompi_proc_t      **proc_array;     /**< list of pointers to ompi_proc_t structures
                                  for each process in the group */
```

Comment:
i do not get it.
a pointer (of pointer of some struct) was replaced by a pointer (of pointer of some struct) e.g. 
```sizeof(struct oshmem_proc_t **) == sizeof (struct ompi_proc_t **)```
so how can this break alignment ?

or are you saying that the padding array of an ```ompi_proc_t``` might not be aligned, and hence the ```oshmem_proc_t``` will not be aligned too ?
i configured with and without ```--enable-debug```, and i could not see any difference.
shall i simply declare the padding of ```ompi_proc_t``` as
```void *padding[OMPI_PROC_PADDING_SIZE/sizeof(void *);```
instead of
```char padding[OMPI_PROC_PADDING_SIZE;```
in order to guarantee it will be aligned ?